### ISSUE TITLE
redundancy with upstream HACKING.adoc?

### ISSUE THREAD

**Issue by gasche** [ocamllabs/ocaml-internals#3]
I wonder if this project is not redundant with the idea of having contributing-friendly stored directly in the upstream repo in the form of [HACKING.adoc](https://github.com/ocaml/ocaml/blob/trunk/HACKING.adoc) pages.

These pages barely existed (they had been unchanged for years) when the current wiki was created, so I'm not blaming anyone for thinking that an outside-the-tree document would be easier to build. But now that it is more active again, I wonder if the content that is being created here would not be better there: I think that being close to the source and versioned is quite helpful -- besides the visibility gains.

**Comment by yallop** [comment]
I agree that `HACKING.adoc` is a better location for this material.  Would you like to move it over (with permission from the contributors)?  Once the content is upstreamed we can delete this repository.

**Comment by gasche** [comment]
It's work to move it over, so I won't do it overnight, but sure, if we agree in principle, let's do it -- gradually.

**Comment by gasche** [comment]
I'll look at who the contributors are for each page and ping them as I go along.
